clio-util/src/main/scala/org/broadinstitute/clio/util/AuthUtil.scala

```diff
@@ -24,25 +25,31 @@ object AuthUtil extends ModelAutoDerivation {
     *
     * @param serviceAccountPath Option of path to the service account json
     */
-  def getAccessToken(serviceAccountPath: Option[Path]): AccessToken = {
-    serviceAccountPath
-      .map { jsonPath =>
-        loadServiceAccountJson(jsonPath)
-      }
-      .fold(
-        new AccessToken(Process("gcloud auth print-access-token").!!.trim, null)
-      )(getCredsFromServiceAccount)
+  def getAccessToken(serviceAccountPath: Option[Path]): Try[AccessToken] = {
```

Comment:
Oops, lost in translation. I didn't want this method to return a `Try`, I wanted `loadServiceAccountJson` to. You should be able to just put `Try` before the opening curly of its body.